test/datatypes.test.js

```diff
@@ -72,7 +72,20 @@ describe('MySQL specific datatypes', function() {
       });
     });
   });
-
+  it('should create a model instance with binary types', function(done) {
+    var str = 'This is a test';
+    var name = 'bob';
+    var bob = {name: name, bin: new Buffer.from(str)};
+    var bm = BlobModel.create(bob, function (err, obj) {
+      assert.ok(!err);
+      assert.equal(obj.bin.toString(), str);
+      BlobModel.findOne({where: {name: name}}, function (err, found) {
```

Comment:
I beleive assert is failing the test in these two cases and done(err) is not needed